### ISSUE TITLE
rally Dockerfile does not take effect from committed change

### ISSUE THREAD

**Issue by huikang** [openvswitch/ovn-scale-test#54]
Problem description: if a commit change anything in rally-ovs, the rally Dockerfile can not receive the new update. This is because rally Dockerfile clones the repo form upstream.

Then a following PR is needed to fix this issue, e.g., #53 .

One possible solution is to generate Dockerfile dynamically, using jinja2 tempate.

**Comment by mestery** [comment]
@huikang Is this still relevant now that #53 has merged?

**Comment by huikang** [comment]
@mestery Yes, this is because rally Dockerfile contains the link to this ovn-scale-test repo. We should fix this later.
But, lets keep this as an open issue so it will not get lost. Thanks.

**Comment by huikang** [comment]
Lets keep this issue open for some time to ensure it works for everyone
